### DISCUSSION TITLE
Traefik with Let's Encrypt causing downtime on renewal?

### DISCUSSION THREAD

**Discussion by SimonVillage** [kube-hetzner/terraform-hcloud-kube-hetzner#187] [Q&A] [answered]
Hi,

from the terraform.tfvars.example file it says:

> however it's not optimal, as the free version of Traefik causes a little bit of downtime when when the certificates get renewed.

I couldn't find any source to this, does someone has a link?

**Comment by mysticaltech** [comment, marked as answer]
@SimonHausdorf Sorry for the late answer; I just saw this now. Here you go https://doc.traefik.io/traefik/v2.0/providers/kubernetes-crd/.

In the examples section of the readme, you will find an example of how to use cert-manager. It's worth it!
